server/.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file needed?

Reply:
It is not and has been deleted! Thank you!